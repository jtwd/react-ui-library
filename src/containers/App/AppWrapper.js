import styled from 'styled-components'
import Wrapper from '../../components/Wrapper'
import { siteMaxWidth } from '../../components/_theme/units'

const AppWrapper = styled(Wrapper)`
  max-width: ${siteMaxWidth};
  min-height: 100vh;
  margin: 0 auto;
`

export default AppWrapper
