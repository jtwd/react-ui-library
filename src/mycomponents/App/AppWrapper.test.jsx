import React from 'react'
import { shallow } from 'enzyme'
import AppWrapper from './AppWrapper'

test('AppWrapper renders correctly', () => {
  const component = shallow(<AppWrapper />)
  expect(component).toMatchSnapshot()
})
