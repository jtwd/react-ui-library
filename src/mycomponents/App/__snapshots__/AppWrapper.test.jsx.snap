// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppWrapper renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <styled.div
    className="sc-bwzfXH kqMkuf"
    innerRef={undefined}
/>,
  "nodes": Array [
    <styled.div
      className="sc-bwzfXH kqMkuf"
      innerRef={undefined}
/>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <Styled(styled.div) />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StyledComponent {
        "_reactInternalInstance": [Circular],
        "attrs": Object {},
        "context": Object {
          "__styled-components-stylesheet__": undefined,
          "__styled-components__": undefined,
        },
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "generatedClassName": "kqMkuf",
          "theme": Object {},
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <styled.div
          className="sc-bwzfXH kqMkuf"
          innerRef={undefined}
/>,
        "_debugID": 2,
        "_renderedOutput": <styled.div
          className="sc-bwzfXH kqMkuf"
          innerRef={undefined}
/>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Styled(styled.div) />,
}
`;
